# Copyright 2023, Advanced Micro Devices, Inc
# SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note
# Downloaded from https://support.xilinx.com/s/article/000035732?language=en_US
From cfe8aafb42533fcedecebe7eda27fc4ecb1b659e Mon Sep 17 00:00:00 2001
From: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
Date: Fri, 6 Oct 2023 11:50:28 +0530
Subject: [LINUX PATCH 3/5] arm64: configs: Enable CONFIG_DRM_XLNX_HDMITX for
 zynqmp and versal

Enable HDMI 2.1 Tx Subsystem driver for zynqmp and versal

Signed-off-by: Rajesh Gugulothu <rajesh.gugulothu@amd.com>
---
 arch/arm64/configs/xilinx_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/xilinx_defconfig b/arch/arm64/configs/xilinx_defconfig
index 5dd99de88b46..64b589a91207 100644
--- a/arch/arm64/configs/xilinx_defconfig
+++ b/arch/arm64/configs/xilinx_defconfig
@@ -450,6 +450,7 @@ CONFIG_DRM_XLNX_SDI=y
 CONFIG_DRM_XLNX_BRIDGE_CSC=y
 CONFIG_DRM_XLNX_BRIDGE_SCALER=y
 CONFIG_DRM_XLNX_BRIDGE_VTC=y
+CONFIG_DRM_XLNX_HDMITX=y
 CONFIG_FB=y
 CONFIG_FRAMEBUFFER_CONSOLE=y
 CONFIG_SOUND=y
-- 
2.17.1

